### PR TITLE
OT260-40 List of the category names

### PR DESCRIPTION
**Pull request by NataliaGatti**
Hi guys! Hope you are well!
I created the route to get the list of category names. 
Then I created the index method in CategoriesController to to return a list of category names in a json format.
In the method I use a new variable (@cat_serializer) to obtain the serialized information because rubocop observed me as too extensive the line of code.
I created the serilizer of model category to get the rigth render of the list. Because of this, I add accessor to the atributtes name and description in the model category.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  namespace :api, defaults: { format: 'json' } do
+  namespace :api, defaults: { format: :json } do
```
Comment (FernandoCiballero) [inline config/routes.rb:2]: You should check the main branch because this change has been updated.

File: config/database.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ default: &default
   username: <%= ENV.fetch('DB_USER') %>
   password: <%= ENV.fetch('DB_PASSWORD') %>
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  host: <%= ENV.fetch('DB_HOST') %>
+  host: <%= ENV.fetch('DB_HOST')  %>
```
Comment (FernandoCiballero) [inline config/database.yml:7]: 👏

File: app/controllers/api/v1/categories_controller.rb [hunk 3]
```diff
@@ -3,6 +3,12 @@
 module Api
   module V1
     class CategoriesController < ApplicationController
+      def index
+        @categories = Category.all
```
Comment (FernandoCiballero) [inline app/controllers/api/v1/categories_controller.rb:7]: You need to check if the user is the admin

File: app/models/category.rb [hunk 4]
```diff
@@ -17,6 +17,7 @@
 #
 class Category < ApplicationRecord
   include Discard::Model
+  attr_accessor :name, :description
```
Comment (FernandoCiballero) [inline app/models/category.rb:20]: What is the purpose of this line.

**Review by FernandoCiballero — CHANGES_REQUESTED** [review]
Left some comments and also you need to rebase your branch.

File: config/routes.rb (hunk 1 above)
Comment (NataliaGatti) [reply config/routes.rb:2]: rebase done. Thnaks!

File: app/models/category.rb (hunk 4 above)
Comment (NataliaGatti) [reply app/models/category.rb:20]: I think I saw it in a guide but rereading the serializer instructions I see I got confused.
I’m already removing the line from the model.
Thanks!

File: app/controllers/api/v1/categories_controller.rb (hunk 3 above)
Comment (NataliaGatti) [reply app/controllers/api/v1/categories_controller.rb:7]: Ok, I added `before_action :admin?, only: :index` to check that the user is the admin

**Pull request merged**